training/new_addresses.xml

```diff
@@ -0,0 +1,12 @@
+<AddressCollection>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreType>Business</StreetNamePreType> <StreetName>190</StreetName> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>13,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreType>business</StreetNamePreType> <StreetName>190</StreetName> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>3,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreType>business</StreetNamePreType> <StreetName>190</StreetName> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>8,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreType>business</StreetNamePreType> <StreetName>190</StreetName> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>42,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreType>business</StreetNamePreType> <StreetName>190</StreetName> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>41,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
+  <AddressString><AddressNumber>1200</AddressNumber> <StreetNamePreDirectional>business</StreetNamePreDirectional> <StreetNamePreType>190</StreetNamePreType> <OccupancyType>ste</OccupancyType> <OccupancyIdentifier>42,</OccupancyIdentifier> <PlaceName>COVINGTON,</PlaceName> <StateName>LA</StateName> <ZipCode>70433</ZipCode></AddressString>
```

Comment:
In the first five training examples, `business` is tagged as `StreetNamePreType` and `190` is tagged as `StreetName`, and I agree that this seems like the proper pattern for this type of address. But here `business` is tagged as `StreetNamePreDirectional` and `190` is `StreetNamePreType`... any reason for the discrepancy?